src/rtString.h

```diff
@@ -115,6 +116,8 @@ class rtString
 
   rtString substring(size_t pos, size_t len = 0) const;
 
+  void toLower() { char *p = mData; while(*p) { *p = tolower(*p); p++; } };
```

Comment:
toLower() won't return the desired result for UTF-8 strings.  Can you please document or rename method to `toLowerAscii()`?

Reply:
Done